injector_test.go

```diff
@@ -408,3 +427,20 @@ func BenchmarkProcess(b *testing.B) {
 		}
 	})
 }
+
+func assertJSONEqual(t *testing.T, expected, actual string) {
```

Comment:
извини, вовремя не заметил, эта функа, по сути, хелпер ассерта.  принято их так и помечать чтобы сообщение об ошибке на нужную строчку в коде указывало:
```suggestion
func assertJSONEqual(t *testing.T, expected, actual string) {
    t.Helper()
```

Reply:
Сделал